# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/win/src/resolver.cc
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 3
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/sandbox/win/src/resolver.cc
--- a/security/sandbox/chromium/sandbox/win/src/resolver.cc	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/sandbox/win/src/resolver.cc	Wed Jun 12 11:10:48 2019 +0100
@@ -18,15 +18,14 @@
                              const void* interceptor_entry_point,
                              void* thunk_storage,
                              size_t storage_bytes) {
-  if (NULL == thunk_storage || 0 == storage_bytes ||
-      NULL == target_module || NULL == target_name)
+  if (!thunk_storage || 0 == storage_bytes || !target_module || !target_name)
     return STATUS_INVALID_PARAMETER;
 
   if (storage_bytes < GetThunkSize())
     return STATUS_BUFFER_TOO_SMALL;
 
   NTSTATUS ret = STATUS_SUCCESS;
-  if (NULL == interceptor_entry_point) {
+  if (!interceptor_entry_point) {
     ret = ResolveInterceptor(interceptor_module, interceptor_name,
                              &interceptor_entry_point);
     if (!NT_SUCCESS(ret))